tests/test_stamp.py

```diff
@@ -257,35 +257,35 @@ def new_toplevel_only(self, object_types):
     # 1. Faint star
     # There aren't any very faint stars in this db file.  The faintest is around flux=2000.
     # But that's easily small enough to hit the minimum stamp size.
-    obj_num = 2618
+    obj_num = 2619
     image, _ = galsim.config.BuildStamp(config, obj_num, logger=logger, do_noise=False)
     print(obj_num, image.center, image.array.shape, image.array.sum())
     assert image.array.shape == (40,40)
     assert 2000 < image.array.sum() < 2300  # 2173
 
     # 2. 10x brighter star.  Still minimum stamp size.
-    obj_num = 2698
+    obj_num = 2699
     image, _ = galsim.config.BuildStamp(config, obj_num, logger=logger, do_noise=False)
     print(obj_num, image.center, image.array.shape, image.array.sum())
     assert image.array.shape == (40,40)
     assert 24000 < image.array.sum() < 27000  # 25593
 
     # 3. 10x brighter star.  Needs bigger stamp.
-    obj_num = 2745
+    obj_num = 2746
     image, _ = galsim.config.BuildStamp(config, obj_num, logger=logger, do_noise=False)
     print(obj_num, image.center, image.array.shape, image.array.sum())
     assert image.array.shape == (106,106)
     assert 250_000 < image.array.sum() < 280_000  # 264459
 
     # 4. 10x brighter star.  (Uses photon shooting, but checks the max sb.)
-    obj_num = 2610
+    obj_num = 2611
     image, _ = galsim.config.BuildStamp(config, obj_num, logger=logger, do_noise=False)
     print(obj_num, image.center, image.array.shape, image.array.sum())
     assert image.array.shape == (350,350)
     assert 2_400_000 < image.array.sum() < 2_700_000  # 2591010
 
     # 5. Extremely bright star.  Maxes out size at _Nmax.  (And uses fft.)
-    obj_num = 2696
+    obj_num = 2697
```

Comment:
Why did all these obj_nums increase by 1?

Reply:
I added a galaxy with the desired properties to the galaxy catalog file.  Since the stars are handled by skyCatalogs after the galaxies, I had to increment each obj_num by 1 in order to have the same stars for these tests.

Reply:
Got it.  That makes sense.